server/game/cards/03_TWI/units/_501StLiberator.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Trait } from '../../../core/Constants';
+
+export default class _501stLiberator extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '7252148824',
+            internalName: '501st-liberator',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addWhenPlayedAbility({
```

Comment:
Can't remember which of these files is correct, can you please remove whichever one is wrong